draft-ietf-sedate-datetime-extended.md

```diff
@@ -358,14 +358,11 @@ This email header convention is in actual use, while its adaptation into
 handicapped by the fact that {{ISO8601-2000}} does not actually allow `-00:00`.
 
 Implementations that needed to express the semantics of `-00:00`
-therefore tended to use `Z` as a "neutral" offset instead.
-
-## Update to RFC 3339
+therefore tended to use `Z` instead.
 
 This specification updates {{Section 4.3 of RFC3339}}, aligning it with the actual
-practice of interpreting the local offset `Z`: this is no longer
-understood to "imply that UTC is the preferred reference point for the
-specified time".
+practice of interpreting the offset `Z` to mean the same as`-00:00`:
```

Comment:
Again, perhaps worth noting RFC 3522 (&c.)'s de facto support of this position.

Reply:
What we came up with does not have any relation to what 5322/5322bis are saying, because they do not use the same date format and therefore do not have the problem that is being solved here.